force-app/main/default/lwc/donationHistoryFilter/__tests__/donationHistoryFilter.test.js

```diff
@@ -0,0 +1,34 @@
+import { createElement } from 'lwc';
```

Comment:
One more scenario to cover in this test class, assert that this component fires the change event when the filter changes.